### PR TITLE
broken link fixes in protocols.md

### PR DESCRIPTION
**Pull request by mikestarrdev**

### CODE REVIEW

File: technology/protocols.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ setTokens([ tokenContractAddress, ... ])
 
 # RequestForQuoteERC20
 
-AirSwap [RequestForQuoteERC20](./glossary.md#request-for-quote-rfq) is a client-server protocol used by market makers running servers from which clients request ERC20 orders via HTTP or WebSocket. In RFQ, the server is the signer (i.e. `signerAmount`, `signerToken`) and the client is the sender (i.e. `senderAmount`, `senderToken`).
+AirSwap [AirSwap RequestForQuoteERC20](./glossary.md#request-for-quote-rfq) is a client-server protocol used by market makers running servers from which clients request ERC20 orders via HTTP or WebSocket. In RFQ, the server is the signer (i.e. `signerAmount`, `signerToken`) and the client is the sender (i.e. `senderAmount`, `senderToken`).
```
Comment (dmosites) [inline technology/protocols.md:61]: Redundant "AirSwap"

File: technology/protocols.md [hunk 2]
```diff
@@ -568,4 +568,4 @@ The server can specify formulas to use for pricing. Each formula is an expressio
 
 **Client wants to swap WETH into `1000` USDT.** Client looks up baseToken USDT and quoteToken WETH and uses the `ask` levels above. `1000` is multiplied by `0.00055` for a total of `0.55` WETH.
 
-**Client wants to swap USDT into `1` WETH.** Client looks up baseToken WETH and quoteToken USDT and uses the `ask` levels above. `1` is multiplied by `2001` for a total of `2001` WETH.
\ No newline at end of file
+**Client wants to swap USDT into `1` WETH.** Client looks up baseToken WETH and quoteToken USDT and uses the `ask` levels above. `1` is multiplied by `2001` for a total of `2001` WETH.
```
Comment (dmosites) [inline technology/protocols.md:571]: What's different here?

Comment (mikestarrdev) [reply technology/protocols.md:571]: I fixed my VScode's prettier settings, and I think it fixed some type of formatting. I don't see a difference, and I'm actually confused by this

File: technology/protocols.md (hunk 1 above)
Comment (mikestarrdev) [reply technology/protocols.md]: Yeah. I think the link was broken because /glossary is 1 directory up. Changed the path

**Review by dmosites — APPROVED** [review]

**Pull request merged**